common/main/en_FR.xml

```diff
@@ -11,4 +11,10 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		<language type="en"/>
 		<territory type="FR"/>
 	</identity>
+	<numbers>
+		<symbols numberSystem="latn">
+			<decimal>,</decimal>
+			<group> </group>
```

Comment:
error here. is this supposed to be a different kind of space? fr.xml has NNBSP

Reply:
It gets collapsed to `<group></group>` or `<group/>` without `xml:space="preserve"` so the parser sees a 0-length group string.  Probably better to fix it here.